tools/ratbagd.py

```diff
@@ -111,7 +111,11 @@ def __init__(self, interface, object_path):
         GObject.GObject.__init__(self)
 
         if _RatbagdDBus._dbus is None:
-            _RatbagdDBus._dbus = Gio.bus_get_sync(Gio.BusType.SYSTEM, None)
+            try:
+                _RatbagdDBus._dbus = Gio.bus_get_sync(Gio.BusType.SYSTEM, None)
+            except GLib.Error:
+                raise RatbagdDBusUnavailable()
+
             if _RatbagdDBus._dbus is None:
```

Comment:
I know it's not part of this PR; I was trying to say that according to the documentation checking for `None` **and** catching the exception isn't necessary.